app/controllers/admin/SettingsController.php

```diff
@@ -135,6 +147,20 @@ public function postEdit()
             $setting->slack_endpoint = e(Input::get('slack_endpoint'));
             $setting->slack_channel = e(Input::get('slack_channel'));
             $setting->slack_botname = e(Input::get('slack_botname'));
+            $setting->ldap_enabled = e(Input::get('ldap_enabled', '0'));
+            $setting->ldap_server = e(Input::get('ldap_server'));
+            $setting->ldap_uname = e(Input::get('ldap_uname'));
+            $setting->ldap_pword = e(Input::get('ldap_pword'));
+            $setting->ldap_basedn = e(Input::get('ldap_basedn'));
+            $setting->ldap_filter = e(Input::get('ldap_filter'));
```

Comment:
The filter I use for testing LDAP (since I don't have access to my own LDAP server) is `&(cn=*)`. Anything with an ampersand is going to be escaped by `e()` method before being inserted into the database, which is going to break the  LDAP integration. Can you pull the escaping from anything that could potentially have special characters in it? We can escape it on the way out when we display it.